dll/win32/msgina/gui.c

```diff
@@ -989,8 +991,22 @@ SecurityDialogProc(
                         EndDialog(hwndDlg, WLX_SAS_ACTION_LOGOFF);
                     return TRUE;
```

Comment:
Here's the code for force-logoff for you. Though seems it would not be handled by winlogon.
```c
                    if (ISKEYDOWN(VK_CONTROL))
                    {
                        EndDialog(hwndDlg, WLX_SAS_ACTION_FORCE_LOGOFF);
                    }
                    else if (OnLogOff(hwndDlg, pgContext) == IDYES)
                    {
                        EndDialog(hwndDlg, WLX_SAS_ACTION_LOGOFF);
                    }
                    return TRUE;
```

Reply:
@cbialorucki noticed that we didn't handle the _FORCE_LOGOFF in winlogon, though.